tests/swoole_curl/symfony-noco.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+swoole_curl: symfony http client
+--SKIPIF--
+<?php
+require __DIR__ . '/../include/skipif.inc';
+?>
+--FILE--
+<?php
+require __DIR__ . '/../include/bootstrap.php';
+require_once TESTS_LIB_PATH . '/vendor/autoload.php';
+
+use Swoole\Runtime;
+use Symfony\Component\HttpClient\HttpClient as SymfonyHttpClient;
+use Symfony\Component\HttpClient\HttplugClient as SymfonyHttplugClient;
+use Http\Client\Common\PluginClient;
+use Http\Discovery\Psr17FactoryDiscovery;
+
+$httpClient = new SymfonyHttplugClient(
```

Comment:
The test could fail when package `symfony/http-client` has a new major release out. Better specify a version constraint for package `symfony/http-client` [here](https://github.com/swoole/swoole-src/blob/v4.8.2/tests/include/lib/composer.json#L25).